octicons/templatetags/octicons.py

```diff
@@ -75,7 +75,8 @@ def size(self):
         }
 
         # if a custom width or height is passed in
-        if self.options.get('width') or self.options.get('height'):
+        if ( (self.options.get('width') or self.options.get('height')) and
```

Comment:
There shouldn't be any spaces around parantheses.
```suggestion
        if ((self.options.get('width') or self.options.get('height')) and
```